.github/ISSUE_TEMPLATE/failing-test.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Failing Test
+about: Report test failures in Kubernetes federation v2 CI jobs
+
+---
+
+<!-- Please only use this template for submitting reports about failing tests in Kubernetes CI jobs -->
+
+**Which jobs are failing**:
+
+**Which test(s) are failing**:
+
+**Since when has it been failing**:
+
+**Testgrid link**:
```

Comment:
Travis results don't end up in testgrid (I think that's a function of running tests with test-infra), so this could be omitted.